lib/logflare/backends/adaptor/postgres_adaptor.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Logflare.Backends.Adaptor.PostgresAdaptor do
```

Comment:
think we should also start documenting how each adaptor works as well, small issue. just a brief description of the ingestion flow works

Reply:
Agreed otherwise it's a lot of code to read